### DISCUSSION TITLE
Vertical slides

### DISCUSSION THREAD

**Discussion by meghprkh** [marp-team/marp#141] [Ideas] [answered]
Add vertical slide support like RevealJS - https://revealjs.com/vertical-slides/

This can be enabled via an additional separator imo

Allows to organize content more effectively imho

**Comment by yhatt** [comment, marked as answer]
Surely vertical slides may be helpful to make effective slides, but Marp team is not taking it on purpose because of its low discoverability for audiences.

> No, there is not vertical slide in Marp because we think 2 way navigation is hard to notice for audience.
> 
> It may resolve by [making plugin for Marpit framework](https://marpit.marp.app/usage?id=extend-marpit-by-plugins), but it would be too hard. Probably continue using reveal.js is the best way if you still want navigation for 2 directions.
>
> _-- https://github.com/yhatt/marp-cli-example/issues/7_

FYI Marpit framework also can use together with reveal.js: https://codesandbox.io/s/nw80vrxvpp
